docs/website/components/Sidebar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="sidenav sticky pt-4 pb-4">
+  <div class="sidenav sticky pb-4">
```

Comment:
I'm not sure I agree with this. I have a TODO to make it such that the bottom of the current doc's title is aligned vertically with the first element of the sidebar. It make the main content pop better IMHO.

Reply:
I've seen this strategy in a number of sites, but I have been using rook's as a reference:
<img width="1281" alt="Screen Shot 2019-10-31 at 2 35 56 PM" src="https://user-images.githubusercontent.com/3383143/67987928-d6bb5b80-fbeb-11e9-91c4-053d3bdfd641.png">

Reply:
Good call, I'll revert.